test/threadstest.c

```diff
@@ -125,6 +126,8 @@ static void rwwriter_fn(int id, int *iterations)
 
     for (count = 0; ; count++) {
         new = CRYPTO_zalloc(sizeof (int), NULL, 0);
+        if (new == NULL)
```

Comment:
Consider:
```suggestion
        if (!TEST_ptr(new))
```
which will produce an error message.